lib/rules/no-invalid-double-slash-comments/README.md

```diff
@@ -1,6 +1,6 @@
 # no-invalid-double-slash-comments
 
-Disallow double-slash comments (`//...`) which are not supported by CSS and [could lead to unexpected results](http://stackoverflow.com/a/20192639/130652).
+Disallow double-slash comments (`//...`) which are not supported by CSS and [could lead to unexpected results](https://stackoverflow.com/a/20192639/130652).
```

Comment:
Ha, it does to, I use the _HTTPS Everywhere_ Chrome extension from EFF.

And for me, it remains on the HTTPS site, so either or I guess

I've a preference to make as many as possible links used HTTPS